docs/reference/aggregations/metrics/tophits-aggregation.asciidoc

```diff
@@ -4,6 +4,9 @@
 A `top_hits` metric aggregator keeps track of the most relevant document being aggregated. This aggregator is intended
 to be used as a sub aggregator, so that the top matching documents can be aggregated per bucket.
 
+TIP: `top_hits` is designed for use in nested aggregations. If you want to group
```

Comment:
`nested` is kind of an overloaded term. Maybe it'd be better to say "If you find yourself writing a `top_hits` aggregation at the top level then you should probably replace it with field collapsing."

Reply:
Thanks for the feedback @nik9000. I reworded this to avoid `nested agg` with e39bb5ee4bdd7da902a2d4ecacb06853d763c3ca. Let me know if that phrasing works for you.